processing/src/main/java/org/apache/druid/segment/virtual/RowBasedExpressionColumnValueSelector.java

```diff
@@ -60,10 +68,16 @@ public RowBasedExpressionColumnValueSelector(
   }
 
   @Override
-  public ExprEval getObject()
+  protected ExprEval<?> eval()
   {
     // check to find any arrays for this row
-    List<String> arrayBindings = unknownColumns.stream().filter(this::isBindingArray).collect(Collectors.toList());
```

Comment:
FYI, the de-streaming here is because I noticed the stream pipeline adding extra CPU time and allocations in profiling.